packages/acot-preset-wcag/docs/rules/focus-dialog.md

```diff
@@ -0,0 +1,61 @@
+# focus-dialog
```

Comment:
I prefer the rule name `dialog-focus` to match the naming conventions of other rules.